content/post/092-launching-testground.md

```diff
@@ -0,0 +1,118 @@
+
+And since not all test scenarios require thousands of instances, you can **run test plans locally as executables or Docker containers,** saving the overhead, time and cost of resorting to cluster deployments for when you need to scale beyond ~300 instances, or so. For those larger cases, **Testground supports Kubernetes out-of-the-box.** 
+
+To illustrate the above, consider a 200-instance test run. With Kubernetes, it could take 1-2 minutes to ship a Docker image to a remote registry, schedule a Kubernetes job, and wait for the workload to start. With the local Docker runner, that overhead is zero, and the run is immediate. It doesn’t sound like much, but developers know the death by a thousand cuts (or context switches) such delays do to one’s productivity and workflow.
+
+The cherry on the top of all this is the **observation pipeline (experimental)**. Test plans can record raw data points or aggregate metrics (histograms, [EWMA](https://en.wikipedia.org/wiki/EWMA_chart), counters, etc.) about the system under test, as well as emit arbitrary output assets. **The Testground platform will harvest them and make them available for one-stop collection.**
+
+![Testground Architecture](/092-launching-testground/launch_testground_architecture.png)
+
+<p style="text-align:center;font-size:75%;"> Architecture diagram, focused on a test plan’s inputs and outputs, highlighting the circuitry of the observability pipeline</p>
+
+## Getting started
+
+Want to hit the ground running? Try the following:
+
+1. Head over to the [Getting started](https://docs.testground.ai/getting-started) section of our docs.
+2. Make sure to check out the [testground/testground](https://github.com/testground/testground)  repo on GitHub.
+3. Take a peek at some [example test plans](https://github.com/testground/testground/tree/master/plans).
+4. Explore the [reference godocs of the Testground SDK](https://pkg.go.dev/github.com/testground/sdk-go?tab=overview), the library you will use to write Testground test plans.
+
+<br/>
+
+## What’s happening next?
+
+The Testground team paces itself in flexible bi-weekly sprints.
+
+In the next couple of weeks, we plan to solidify the foundations of Testground by boosting test coverage across the codebase. We affectionately call this phase “test the testground” 😃
+
+At the tail end of that sprint, we will produce v0.7 of the Testground Core, which will serve as the plumbing for the next features we plan to land:
+
+* time-shared cluster deployments.
+* tight continuous integration with GitHub (so that developers can @mention Testground and have their PRs automatically tested).
+* better matrix-based forms to visualize test run results.
+
+<br/>
+Needless to say, you’re welcome to pitch in at any point in time! Continue reading to learn how.
+
+![Testground flowchart](/092-launching-testground/launch_testground_flowchart.png)
```

Comment:
```suggestion
![Testground architecture diagram](/092-launching-testground/launch_testground_flowchart.png)
```

Reply:
Change the filename? It may get better hits on Google Images if we name it "testground_architecture.png"?

Reply:
Ignore my commends, this was the wrong image :D